index.js

```diff
@@ -0,0 +1,79 @@
+console.log("Hello world");
+//task one
+let yearOfBirth = 1989;
+let yearFuture=2027;
+let age = yearFuture - yearOfBirth;
+console.log(`You will be  ${age}  years old in  ${yearFuture}  .`);
+//task two
+let dogYearOfBirth = 2015;  
+let dogYearFuture = 2027;   
+let dogYear = dogYearFuture - dogYearOfBirth; // Calculate the age in human years
+
+// Step 2: Add the extra variable
+let shouldShowResultInDogYears = true;  // Change this to false to show result in human years
+
+// Step 3: Calculate the age in dog years if needed
+let ageInDogYears = dogYear * 7;
+
+// Step 4: Log the result based on the value of shouldShowResultInDogYears
+if (shouldShowResultInDogYears) {
+  console.log("Your dog will be " + ageInDogYears + " dog years old in " + dogYearFuture + ".");
```

Comment:
This can look tidier with [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - the same way as you've used in the first task